DevTchap/Config/BuildSettings.swift

```diff
@@ -97,7 +97,7 @@ final class BuildSettings: NSObject {
     static let serverUrlPrefix = "https://matrix."
     static let preferredIdentityServerNames = [
         "dev01.tchap.incubateur.net",
-        "dev02.tchap.incubateur.net"
+//        "dev02.tchap.incubateur.net"
```

Comment:
You should support this homeserver

Reply:
Pas encore : ProConnect n'était pas dispo sur dev02.